psychopy/app/builder/builder.py

```diff
@@ -1,14 +1,15 @@
-# Part of the PsychoPy library
-# Copyright (C) 2015 Jonathan Peirce
-# Distributed under the terms of the GNU General Public License (GPL).
+"""
+Defines the behavior of Psychopy's Builder view window
+Part of the PsychoPy library
+Copyright (C) 2015 Jonathan Peirce
+Distributed under the terms of the GNU General Public License (GPL).
+"""
 
 from __future__ import absolute_import
 from __future__ import print_function
 from __future__ import division
 
-from builtins import str
```

Comment:
Are you certain about this change?
http://python-future.org/compatible_idioms.html#basestring